src/mixins/filing-template-mixin.ts

```diff
@@ -124,7 +126,8 @@ export default class FilingTemplateMixin extends Vue {
     this.setOrgPersonList(draftFiling.incorporationApplication.parties)
 
     // Set Share Structure
-    this.setShareClasses(draftFiling.incorporationApplication.shareClasses)
+    this.setShareClasses(draftFiling.incorporationApplication.shareStructure
+      ? draftFiling.incorporationApplication.shareStructure.shareClasses : [])
```

Comment:
This is OK but you could also do `x || []`. FYI

Reply:
Specifically, `draftFiling.incorporationApplication.shareStructure?.shareClasses : []`.